### DISCUSSION TITLE
Issues as contributions to notable-like plugin

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#697] [Ideas]
Bug reports & feature ideas are valuable contributions.  It would be useful to summarise them similar to the notable plugin. 

Often the issue contributions will have significant overlap with the commit contributions of the notable plugin, but I think it is better as a separate plugin as it would be very hard to visually merge both concepts.

The queries are different for issues, and I expect most people would want to use different filters and thresholds for issue contributions.

**Comment by lowlighter** [comment]
Yeah, I think it's a good idea too 👍  

Though I'm torn on making a separate plugin, it may be easier to integrate it with the current notable plugin, but add a new badge for issues instead. The queries might be different, but the logic doesn't really change so it might be worth it to keep it in the same plugin (especially since it'd probably be an `indepth` feature because of the need of the search api).

Since #651 has not been done yet, I guess different filters and thresholds for issue contributions would be feasable too

Also, a separate plugin may "duplicate" visually with a lot of redundancy depending on whether one open issues, prs or both

**Comment by lowlighter** [comment]
This was shipped in [`🎩 notable`](https://github.com/lowlighter/metrics/blob/master/source/plugins/notable/README.md) with [`plugin_notable_types`](https://github.com/lowlighter/metrics/blob/master/source/plugins/notable/README.md#plugin_notable_types) and will be available for next release 👍
